package.json

```diff
@@ -57,6 +57,7 @@
   "devDependencies": {
     "@changesets/cli": "^2.26.2",
     "@next/bundle-analyzer": "^13.2.4",
+    "@types/node": "^20.11.16",
```

Comment:
This is unnecessary at the monorepo root `package.json`. We should remove it, and if still needed, move this to the `packages/faustwp-cli/package.json`

Reply:
Just tested this, and it is not needed. Great call, removed.